spec/controllers/openid_token_proxy/callback_controller_spec.rb

```diff
@@ -36,7 +36,11 @@
       context 'with no-op token acquirement hook' do
         it 'redirects to root' do
           OpenIDTokenProxy.configure_temporarily do |config|
-            config.token_acquirement_hook = proc { }
+            block = double('block')
+            config.token_acquirement_hook = proc { |token|
+              block.run(token)
+            }
+            expect(block).to receive(:run).with(instance_of(OpenIDTokenProxy::Token))
```

Comment:
Here's the documentation: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/yield-matchers

It's the `expect do |probe| ... end.to yield_with_args(...)` version we could use here I think.